hw/ip/uart/data/uart.hjson

```diff
@@ -16,8 +16,8 @@
       desc: "raised if the transmit FIFO is past the high-water mark."}
```

Comment:
could you also update the register description and the section "tx_watermark / rx_watermark" in uart/doc/_index.md

Reply:
added, thanks for the reminder.